<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>Assets Browser</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>600</width>
      <height>540</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tabBG">
     <attribute name="title">
      <string>BG</string>
     </attribute>
     <widget class="QColumnView" name="columnViewBG">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>560</width>
        <height>490</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="resizeGripsVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCH">
     <attribute name="title">
      <string>CH</string>
     </attribute>
     <widget class="QColumnView" name="columnViewCH">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>560</width>
        <height>490</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="resizeGripsVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFX">
     <attribute name="title">
      <string>FX</string>
     </attribute>
     <widget class="QColumnView" name="columnViewFX">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>560</width>
        <height>490</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="resizeGripsVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabHelp">
     <attribute name="title">
      <string>Help</string>
     </attribute>
     <widget class="QFrame" name="frameHelp">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>10</y>
        <width>560</width>
        <height>490</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>560</width>
         <height>490</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Assets Browser v0.1.0&lt;br/&gt;Last updated: 30 July 2017&lt;/p&gt;&lt;p&gt;Placeholder text. Need to figure out a way to read in external HTML file to keep the Python script lean for easier coding/debugging.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushBtnNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Asset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxDebug">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Show Debug Window</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelProject">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Project:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCredits">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Huey Yeng Â© 2017&lt;br/&gt;http://taukeke.com&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionAlwaysOnTop"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About Assets Browser</string>
   </property>
  </action>
  <action name="actionAlwaysOnTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always on &amp;Top</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
